### PR TITLE
🐛Source Looker: Fix schema transformation issue

### PR DESCRIPTION
**Pull request by lazebnyi**
## What
Dynamically generated schema hasn't type `null` for some fields, but could be has `None` as value.  

## How
Added `null` to schema type using recursion. 

## Pre-merge Checklist
<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub and connector version bumped by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)

</details>

### CODE REVIEW

File: airbyte-integrations/connectors/source-looker/source_looker/streams.py [hunk 1]
```diff
@@ -251,10 +251,40 @@ def get_parent_endpoints(self) -> List[SwaggerParser.Endpoint]:
     def name(self) -> str:
         return self._name
 
+    @classmethod
+    def format_null_in_schema(cls, schema: Mapping[str, Any]):
```
Comment (davydov-d) [inline airbyte-integrations/connectors/source-looker/source_looker/streams.py:255]: could you add a small unit test for this please?

**Review by davydov-d — CHANGES_REQUESTED** [review]

**Review by davydov-d — APPROVED** [review]

File: airbyte-integrations/connectors/source-looker/source_looker/streams.py (hunk 1 above)
Comment (lazebnyi) [reply airbyte-integrations/connectors/source-looker/source_looker/streams.py:255]: updated

File: airbyte-integrations/connectors/source-looker/source_looker/spec.json [hunk 2]
```diff
@@ -25,7 +25,8 @@
       "client_secret": {
         "title": "Client Secret",
         "type": "string",
-        "description": "The Client Secret is second part of an API3 key."
+        "description": "The Client Secret is second part of an API3 key.",
+        "airbyte_secret": true
```
Comment (girarda) [inline airbyte-integrations/connectors/source-looker/source_looker/spec.json:29]: yikes. good catch

File: airbyte-integrations/connectors/source-looker/unit_tests/unit_test.py [hunk 3]
```diff
@@ -2,7 +2,11 @@
 # Copyright (c) 2022 Airbyte, Inc., all rights reserved.
 #
 
+from source_looker.streams import LookerStream
 
-def test_example_method() -> None:
-    assert True
-    return
+
+def test_format_null_in_schema():
```
Comment (girarda) [inline airbyte-integrations/connectors/source-looker/unit_tests/unit_test.py:8]: can you add a test case that uses the recursion?

**Review by girarda — APPROVED** [review]
lgtm. just a small request for an additional test case

File: airbyte-integrations/connectors/source-looker/unit_tests/unit_test.py (hunk 3 above)
Comment (lazebnyi) [reply airbyte-integrations/connectors/source-looker/unit_tests/unit_test.py:8]: updated

**Pull request merged**